tests/test_framework/phpbb_ui_test_case.php

```diff
@@ -158,7 +158,23 @@ public function getDriver()
 
 	public function visit($path)
 	{
-		$this->getDriver()->get(self::$root_url . $path);
+		// Retry three times on curl issues, e.g. timeout
```

Comment:
I would do something like this to avoid repeating the same 3 times

```php
$attemps = 0;
$retries = 3;

while(true) {
	$attemps++;
	try {
		$this->getDriver()->get(self::$root_url . $path);
		break;
	} catch(Facebook\WebDriver\Exception\WebDriverCurlException $exception) {
		if($attemps == $retries) {
			throw $exception;
		}
	}
}
```

Reply:
We can do this in case it actually works. Was aiming for better debuggability in case it errors. ;)